src/inference/tests/functional/caching_test.cpp

```diff
@@ -2255,7 +2255,9 @@ TEST_P(CachingTest, LoadAUTO_OneDeviceNoImportExport) {
     EXPECT_CALL(*mockPlugin, compile_model(_, _, _)).Times(m_remoteContext ? 2 : 0);
     EXPECT_CALL(*mockPlugin, compile_model(A<const std::shared_ptr<const ov::Model>&>(), _))
         .Times(!m_remoteContext ? 2 : 0);
-    EXPECT_CALL(*mockPlugin, OnCompileModelFromFile()).Times(m_type == TestLoadType::EModelName ? 2 : 0);
+    // AUTO will always read the model from the file before compilation on the target device if it does not support
```

Comment:
Why is AUTO has to do it, if there is no cache support the compile model will read model first then compile.
There is no need to read model by AUTO.